inputFiles/wavePropagation/acous3D_vti.xml

```diff
@@ -0,0 +1,229 @@
+<?xml version="1.0" ?>)
+<Problem>
+
+  <!-- hexahedral mesh generated internally by GEOSX -->
+  <Mesh>
+    <InternalMesh
+      name="mesh"
+      elementTypes="{ C3D8 }"
+      xCoords="{ 0,101}"
+      yCoords="{ 0,101 }"
+      zCoords="{ 0,101 }"
+      nx="{ 5 }"
+      ny="{ 5 }"
+      nz="{ 5 }"
+      cellBlockNames="{ cb }"/>
+  </Mesh>
+
+  <Geometry>
+    <Box
+      name="zpos"
+      xMin="{ -0.1, -0.1, 100.9}"
+      xMax="{ 101.1, 101.1, 101.1}"/>
+    <Box
+      name="zneg"
+      xMin="{ -0.1, -0.1, -0.1}"
+      xMax="{ 101.1, 101.1, 0.1}"/>
+    <Box
+      name="xpos"
+      xMin="{ 100.9, -0.1, -0.1}"
+      xMax="{ 101.1, 101.1, 101.1}"/>
+    <Box
+      name="xneg"
+      xMin="{ -0.1, -0.1, -0.1}"
+      xMax="{ 0.1, 101.1, 101.1}"/>
+    <Box
+      name="ypos"
+      xMin="{ -0.1, 100.9, -0.1}"
+      xMax="{ 101.1, 101.1, 101.1}"/>
+    <Box
+      name="yneg"
+      xMin="{ -0.1, -0.1, -0.1}"
+      xMax="{ 101.1, 0.1, 101.1}"/>
+  </Geometry>
+
+    <!-- The free surface condition the domain -->
+  <FieldSpecifications>
+    <FieldSpecification
+      name="zposFreeSurface"
+      objectPath="faceManager"
+      fieldName="FreeSurface"
+      scale="0.0"
+      setNames="{ zpos }"/>
+    <FieldSpecification
+      name="znegBottomSurface"
+      objectPath="faceManager"
+      fieldName="BottomSurface"
+      scale="0.0"
+      setNames="{ zneg }"/>
+    <FieldSpecification
+      name="LateralFreeSurface"
+      objectPath="faceManager"
+      fieldName="LateralSurface"
+      scale="0.0"
+      setNames="{ xpos, xneg, ypos, yneg }"/>
+    <FieldSpecification
+      name="Epsilon"
+      initialCondition="1"
+      objectPath="ElementRegions/Region/cb"
+      fieldName="epsilon"
+      scale="0.24"
+      setNames="{ all }"/>
+    <FieldSpecification
+      name="Delta"
+      initialCondition="1"
+      objectPath="ElementRegions/Region/cb"
+      fieldName="delta"
+      scale="0.1"
+      setNames="{ all }"/>
+    <FieldSpecification
+      name="f"
+      initialCondition="1"
+      objectPath="ElementRegions/Region/cb"
+      fieldName="f"
+      scale="1.0"
+      setNames="{ all }"/>
+    </FieldSpecifications>
+  <Solvers>
+    <!-- define the solver -->
+    <!-- define the source coordinates -->
+    <!-- define the time source frequency -->
+    <!-- define the receiver coordinates -->
+    <AcousticSEM_VTI
```

Comment:
I am not sure that a name with underscores fits within the GEOS rules for xml tags. Maybe use AcousticVTISEM? Or AcousticVerticalTransverseIsotropicSEM? @acitrain @Bertbk do you have any preference?

Reply:
Just remove the underscore should be fine, the second name is way to long I think..

Reply:
OK done